frontend/src/i18n/fr.json

```diff
@@ -0,0 +1,37 @@
+{
+  "hero_banner": {
+    "title": "Crée des histoires illustrées incroyables avec l'IA !",
+    "subtitle": "Libère ta créativité et donne vie à tes histoires avec des images générées par l'IA.",
+    "start-button": "Commence maintenant"
+  },
+  "sub_banner": {
+    "title": "Libère ta créativité sans effort !",
```

Comment:
Est-ce que l'espace avant le point d'exclamation est par exprès?

Reply:
Not at all! 😅 Will submit this fix through another PR. By the way, I might do more adjustments into the translation file after doing user testing with my kids, etc.